From fa0e88412c85739db154e1929588d012c8b4d9ff Mon Sep 17 00:00:00 2001
From: Iurii Artemenko <iurii_artemenko@epam.com>
Date: Tue, 26 Nov 2019 12:18:03 +0000
Subject: [PATCH] avoid GCC-8 warnings treated as errors

Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 99ce8c2..a2a1d57 100644
--- a/Makefile
+++ b/Makefile
@@ -46,14 +46,14 @@ PKG_CONFIG ?= pkg-config
 VERSION = $(or $(shell test -d .git && git describe --tags), $(shell cat VERSION))
 LIB_CFLAGS   = $(CFLAGS)
 LIB_LDFLAGS  = $(LDFLAGS)
-LIB_CFLAGS  += -Wall -Wextra -Werror -g -fstack-protector-all -Isrc -fPIC
+LIB_CFLAGS  += -Wall -Wextra -Werror -g -fstack-protector-all -Isrc -fPIC -Wno-error=stringop-overflow
 LIB_CFLAGS  += -DVERSION=\"${VERSION}\"
 LIB_LDFLAGS +=
 
 BIN_CFLAGS   = $(CFLAGS)
 BIN_LDFLAGS  = $(LDFLAGS)
 BIN_CFLAGS  += -DVERSION=\"${VERSION}\"
-BIN_CFLAGS  += -Wall -Wextra -Werror -g -fstack-protector-all -Isrc
+BIN_CFLAGS  += -Wall -Wextra -Werror -g -fstack-protector-all -Isrc -Wno-error=stringop-overflow
 BIN_CFLAGS  += $(shell ${PKG_CONFIG} --cflags libxml-2.0)
 BIN_LDFLAGS += $(shell ${PKG_CONFIG} --libs libxml-2.0)
 BIN_LDFLAGS += -L. -lsja1105
-- 
2.7.4

